src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMFileCopier.java

```diff
@@ -0,0 +1,805 @@
+
+    public Description getDescription() {
+        return DESC;
+    }
+
+    @Override
+    public String copyFileStream(ExecutionContext context, InputStream inputStream, INodeEntry node, String destination) throws FileCopierException {
+        ConnectionOptions options = null;
+        String logprompt = "[" + SERVICE_PROVIDER_TYPE + ":" + node.extractHostname() + "] ";
+
+        if (null == context.getExecutionListener()) {
+            System.out.println(logprompt + " Bad plugin context!  NULL ExecutionListener");
+        }
+
+        context.getExecutionListener().log(Constants.VERBOSE_LEVEL, logprompt);
+
+        try {
+            options = willUseConnectionOptions(new ConnectionOptionsBuilder(context, node, framework).build());
+        } catch (ConfigurationException e) {
+            context.getExecutionListener().log(Constants.ERR_LEVEL, logprompt + e.getMessage());
+            throw new FileCopierException(e.getMessage(), StepFailureReason.ConfigurationFailure);
+        }
+
+        context.getExecutionListener().log(Constants.VERBOSE_LEVEL, logprompt + options);
+
+        int result = -1;
+        try {
+            final OverthereConnection connection = new CifsConnectionBuilder(CifsConnectionBuilder.CIFS_PROTOCOL, options,
+                    new DefaultAddressPortMapper()).connect();
+
+            try {
+                OverthereFile dst;
+                if (destination != null && !destination.isEmpty()) {
+                    dst = connection.getFile(destination);
+                    context.getExecutionListener().log(Constants.VERBOSE_LEVEL, logprompt + "Destination file acquired " + dst.getPath());
+                } else {
+                    dst = connection.getTempFile("RD_WINRM_TMP");
+                    context.getExecutionListener().log(Constants.VERBOSE_LEVEL, logprompt + "Temporary file acquired " + dst.getPath());
+                }
+                context.getExecutionListener().log(Constants.VERBOSE_LEVEL, logprompt + "Destination file acquired " + dst.getPath());
+
+                try {
+                    OutputStream w = dst.getOutputStream();
+                    w.write(IOUtils.toByteArray(inputStream));
+                    w.close();
+                } catch (IOException ioe) {
+                    String message = null;
+                    if (context.getLoglevel() > 2) {
+                        ioe.printStackTrace(System.err);
+                        message = ioe.getMessage();
+                    } else {
+                        message = "WinRM Error: " + ioe.getMessage();
+                    }
+                    context.getExecutionListener().log(Constants.ERR_LEVEL, logprompt + "failed: " + message);
+                    throw new FileCopierException(message, Reason.IOWriteError);
+                }
+                context.getExecutionListener().log(Constants.VERBOSE_LEVEL, logprompt + "File copied at: " + dst.getPath());
+                System.out.println(dst.getPath());
```

Comment:
Remove, or use .log() instead of println()

Reply:
Removed